app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Action/Attribute/Tab/Inventory.php

```diff
@@ -88,6 +101,22 @@ public function getDefaultConfigValue($field)
         return $this->stockConfiguration->getDefaultConfigValue($field);
     }
 
+    /**
+     * Returns min_sale_qty configuration for the ALL Customer Group
+     *
+     * @return int
+     */
+    public function getDefaultMinSaleQty()
+    {
+        $default = $this->stockConfiguration->getDefaultConfigValue('min_sale_qty');
+        if (!is_numeric($default)) {
+            $default = $this->serializer->unserialize($default);
+            $default = isset($default[GroupInterface::CUST_GROUP_ALL]) ? $default[GroupInterface::CUST_GROUP_ALL] : 1;
+        }
+
+        return (int) $default;
```

Comment:
@orlangur @pmclain How about the option when qty uses decimals?

Reply:
@valdislav thanks https://github.com/magento/magento2/pull/19095/commits/b3a40584b3bf53a9d2dd06340f6b820563a4eba3

Reply:
@valdislav Please let me know if any additional changes are requested.